src/Nethermind/Nethermind.Serialization.Rlp/RlpBehaviors.cs

```diff
@@ -18,6 +18,7 @@ public enum RlpBehaviors
                                 // when we're calculating tx hash or sending raw transaction we should skip this wrapping
                                 // with additional wrapping for typed transactions we're decoding Uint8Array([TransactionType, TransactionPayload]
                                 // without wrapping we're decoding (TransactionType || TransactionPayload)
+        DisableLazyHash = 64, // Disable lazy hash calculation in transaction
```

Comment:
We would prefer to remove RlpBehaviors not add to them, they are messy can you used some decoder instance with just set property?